File: samples/CS/03-MenuDropDown/RibbonItems.cs
using System;
using System.Windows.Forms;

namespace WinForms.Ribbon
{
    partial class RibbonItems
    {
        public void Init()
        {
            ButtonDropB.Click += new EventHandler<EventArgs>(_buttonDropB_ExecuteEvent);
        }

        void _buttonDropB_ExecuteEvent(object sender, EventArgs e)
        {
            MessageBox.Show("drop B button pressed");
        }

        public void Load()
        {
        }

    }
}

File: samples/VB/03-MenuDropDown/RibbonItems.vb
Imports System

Namespace WinForms.Ribbon

    Partial Class RibbonItems

        Public Sub Init()
            AddHandler ButtonDropB.Click, AddressOf _buttonDropB_ExecuteEvent
        End Sub

        Private Sub _buttonDropB_ExecuteEvent(ByVal sender As Object, ByVal e As EventArgs)
            MessageBox.Show("drop B button pressed")
        End Sub

    End Class
End Namespace
